### ISSUE TITLE
Running the app with react-native cli has a black screen

### ISSUE THREAD

**Issue by special-character** [expo/expo#10904]
### Summary of Issue 
When running the app with react-native start, my app doesn’t get past a black screen. It starts up with my launch screen but changes to a back screen as soon as the green loading bar starts. After loading gets to 100%, the green bar goes away and the app stays black even though I can see a console log from the root App.tsx file.

![CleanShot 2020-11-02 at 14 14 33](https://user-images.githubusercontent.com/5407471/97928088-2363d600-1d1b-11eb-93a8-3b61997224ae.gif)

### Environment - output of `expo diagnostics` & the platform(s) you're targeting
```
  Expo CLI 3.25.1 environment info:
    System:
      OS: macOS 10.15.7
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 12.18.3 - ~/.nvm/versions/node/v12.18.3/bin/node
      Yarn: 1.21.1 - ~/.yarn/bin/yarn
      npm: 6.14.6 - ~/.nvm/versions/node/v12.18.3/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: 12.1/12A7403 - /usr/bin/xcodebuild
    npmPackages:
      expo: ~39.0.2 => 39.0.3 
      react-native: ~0.63.3 => 0.63.3 
      react-native-web: ~0.13.12 => 0.13.18 
```

### Reproducible Demo
https://github.com/special-character/without-expo-client
 
### Expected Behavior vs Actual Behavior
I would expect the app to run with the normal splash screen that it has when running with expo.

I have a hunch that this is related to expo splash screen but not sure. The app loads up just fine with expo start and runs just fine with a production build. Some things I have done to debug:

- Removed everything from App.tsx except a View (and console log)
- Ran expo init for a new bare test app and used the same versions it had for all the starting packages
- Matched the metro.config, babel.config, and app.json with the bare test app
- Reset the sim
- Cleaned the xcode build folder
- Restarted my machine (swinging for the fences on this one)
- Copied the SplashScreen, SplashScreenBackground, and SplashScreen.storyboard from the bare test app
- Tested a physical device

Has anyone run into this or have any more suggestions on how to debug?

**Comment by brentvatne** [comment]
hi there! there isn't much we can do to help without a reproducible example / more info. you might want to try reverting your app to the last known working commit and then slowly adding commits back until the issue reproduces again, then you can look at the individual changes from that commit to find more information.

**Comment by special-character** [comment]
I am working on making a repro but haven't been successful yet. I don't have previous working commits because this is effectively the first time I have run using the `react-native` cli (has been working with expo client app just fine).

Thanks for taking a look anyway.

**Comment by special-character** [comment]
@brentvatne Not sure if you would be willing to take a look at this repo that reproduces the issue I am having. I tried to make it as small as possible:

https://github.com/special-character/without-expo-client

**Comment by brentvatne** [comment]
i cloned and tried to run that project but it doesn't build:

```
CompileStoryboard /Users/brentvatne/code/slap/ios/slap/SplashScreen.storyboard (in target 'Gather' from project 'Gather')
    cd /Users/brentvatne/code/without-expo-client/ios
    export XCODE_DEVELOPER_USR_PATH\=/Applications/Xcode.app/Contents/Developer/usr/bin/..
    /Applications/Xcode.app/Contents/Developer/usr/bin/ibtool --errors --warnings --notices --module gather --output-partial-info-plist /Users/brentvatne/Library/Developer/Xcode/DerivedData/Gather-aawwkdhnfparfdcvwreozycphwro/Build/Intermediates.noindex/Gather.build/Debug-iphonesimulator/Gather.build/SplashScreen-SBPartialInfo.plist --auto-activate-custom-fonts --target-device iphone --target-device ipad --minimum-deployment-target 10.0 --output-format human-readable-text --compilation-directory /Users/brentvatne/Library/Developer/Xcode/DerivedData/Gather-aawwkdhnfparfdcvwreozycphwro/Build/Intermediates.noindex/Gather.build/Debug-iphonesimulator/Gather.build /Users/brentvatne/code/slap/ios/slap/SplashScreen.storyboard
error: Build input file cannot be found: '/Users/brentvatne/code/slap/ios/slap/SplashScreen.storyboard' (in target 'Gather' from project 'Gather')
```

**Comment by special-character** [comment]
Sorry about that, I didn't realize pulling it from another project would copy the references. I will work on fixing those.

**Comment by special-character** [comment]
Thanks for taking a look at that, I fixed those references so that they reference the correct local files.

**Comment by special-character** [comment]
I might have found something interesting at least. In my effort to make everything exactly the same as the working test app I have, I copied the `info.plist` over and it was finally able to show the initial app screen. Looks like this config is the culprit:

```
<key>UIApplicationSceneManifest</key>
	<dict>
		<key>UIApplicationSupportsMultipleScenes</key>
		<true/>
	</dict>
```

Not sure where I got that or why it breaks things but I am guessing I have something configured incorrectly.
